File: app/src/main/java/com/example/concours_passerelle/EmailVerificationActivity.java
package com.example.concours_passerelle;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;

public class EmailVerificationActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_email_verification);

        // Initialiser le bouton "Verify"
        Button btnVerify = findViewById(R.id.btnVerify);

        // Configurer l'action au clic du bouton
        btnVerify.setOnClickListener(v -> {
        // Démarrer l'activité de félicitations
        startActivity(new Intent(EmailVerificationActivity.this, CongratulationsActivity.class));
    });
    }
}

File: app/src/main/java/com/example/concours_passerelle/EmailVerificationActivity.kt
package com.example.concours_passerelle

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class EmailVerificationActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_email_verification)

        val btnVerify: Button = findViewById(R.id.btnVerify)

        btnVerify.setOnClickListener {
            startActivity(Intent(this, CongratulationsActivity::class.java))
        }
    }
}
